windows/Dockerfile

```diff
@@ -23,10 +23,17 @@ ENV WINDOWS_VERSION=${WINDOWS_VERSION:-1809}
 ARG TARGET_ARCH
 ENV TARGET_ARCH=${TARGET_ARCH:-x64}
 
+# Chocolatey package versions
+ENV GIT_VERSION "2.26.2"
+ENV SEVENZIP_VERSION "19.0"
+ENV VS2017BUILDTOOLS_VERSION "15.9.23.0"
+ENV VCPYTHON27_VERSION "9.0.0.30729"
```

Comment:
Question: why is this not added as a label but the rest are?

Reply:
Because I forgot it 😄